docs/preparing_your_paper.md

```diff
@@ -0,0 +1,34 @@
+Preparing your paper
+==========================
+
+## Basics
+
+- How the paper is being produced (Pandoc etc.)
+- How authors can test their submission (what Pandoc command to run)
+  - https://gist.github.com/arfon/09a11047de1c972e37f77e95d2c10efa
+- Iterating with Whedon
+
+## Example paper
+
+- Basic paper structure
+- adrn paper
+
+## Citations
+
+- Show how to create them.
```

Comment:
e.g. link to http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html